src/gui/qgsattributeform.cpp

```diff
@@ -223,14 +223,15 @@ void QgsAttributeForm::setMode( QgsAttributeForm::Mode mode )
   emit modeChanged( mMode );
 }
 
-void QgsAttributeForm::changeAttribute( const QString &field, const QVariant &value )
+void QgsAttributeForm::changeAttribute( const QString &field, const QVariant &value, const QString &toolTip )
 {
   Q_FOREACH ( QgsWidgetWrapper *ww, mWidgets )
   {
     QgsEditorWidgetWrapper *eww = qobject_cast<QgsEditorWidgetWrapper *>( ww );
     if ( eww && eww->field().name() == field )
     {
       eww->setValue( value );
+      eww->widget()->setToolTip( toolTip );
```

Comment:
Sorry, I totally misinterpreted your last message, forget my last message.

What do you think about replacing the placeholder text (on the widget, not the wrapper) temporarily in this case while setting the content to empty?